### DISCUSSION TITLE
New project website: datasette.io

### DISCUSSION THREAD

**Discussion by simonw** [simonw/datasette#1140] [General]
Launched today, the project finally has an official website that's not just the documentation!

https://datasette.io/

There's lots more I want to do with it, but it's complete enough to share with the world.

**Comment by nattaylor** [comment]
The new site is excellent.  I think it might be worth adding something like this on the homepage after the "try a demo" paragraph:

> If you have data to share, [getting started](https://docs.datasette.io/en/stable/getting_started.html) with dataset is easy by either running your own instance or publishing to a cloud provider.
